topics/howtosim.rst

```diff
@@ -0,0 +1,255 @@
+.. _howtosim:
+
+How to run your own N-body simulation in EXP
+###############################################
+
+This tutorial will walk you through the steps to get started with running
+your own N-body simulations in EXP. EXP uses YAML configuration files, and
+we provide an :ref:`an annotated example <yamlconfig>` 
+for reference as well as an :ref:`introduction to YAML syntax <yaml-primer>`. 
+We will also need body and cache files, which we will discuss in more
+detail below. These files can be generated within EXP, or you can provide
+your own body file.
+
+The general steps for creating an N-body simulation in EXP are to:
+
+#. Generate a model file and body files. The model file contains the initial density distribution for your dark matter halo. The body file contains the initial phase space positions for (e.g.) your stellar disk and dark matter halo
+
+#. Generate a cache file with your bases. In this example, you would need one cache file for your cylindrical (disk) basis and another for your spherical (halo) basis  
```

Comment:
Two comments here:
1. I wonder if this conflates two different processes: generating ICs and running `exp`?  E.g. one could bring ICs from a different IC generator.  So I suggest that this 3-step process describes my/our recommendation for generating ICs.
2. I'm unsure whether cache generation deserves it's own step.  True, the basis is generated by `gendisk`.  But it would also be generated by `exp` itself given the force configuration parameters.

Reply:
1) good point - would splitting this page into /simulations/ and /ICs/ be sufficient, or should they be fully separate pages? 
2) This was my misunderstanding as to what was needed! `cache` files are used everywhere in EXP/pyEXP and on first read-through I had no idea where they were supposed to come from, so it would be helpful to have info somewhere on how to generate them

Reply:
Initially, the SL basis cache files were intended to be hidden from the user.  Rather, they were intended to save wall clock time to prevent `exp` from recomputing the basis from the same parameters.  And they were given the hidden names (in Unix) beginning with `.`.  But we found ourselves wanting to use explicitly use these cache files to ensure that the bases were the same between runs or between `gendisk` and `exp`.  At that point, we changed the internal policies to require that the user specify cache file names and not make them hidden files.

In this sense, our use of `cache` has changed.  I suppose that they still are  _caches_ but they are explicit rather than hidden and no longer only a compute-time saving trick for `exp`.